frontend/src/components/FormInput/index.tsx

```diff
@@ -1,10 +1,10 @@
-import { TextField } from "@mui/material";
+import { FormControlLabel, TextField } from "@mui/material";
```

Comment:
@ASpiteri-BCGov   please just have a look, think all is ok, second point of view would be great.